R/fit.R

```diff
@@ -752,13 +754,15 @@ CmdStanFit$set("public", name = "return_codes", value = return_codes)
 #' }
 #'
 profiles <- function() {
-  profiles <- list()
-  i <- 1
-  for (f in self$profile_files()) {
-    profiles[[i]] <- data.table::fread(f, data.table = FALSE)
-    i <- i + 1
-  }
-  profiles
+  if (is.null(private$profiles_)) {
+    private$profiles_ <- list()
+    i <- 1
+    for (f in self$profile_files()) {
+      private$profiles_[[i]] <- data.table::fread(f, integer64 = "character", data.table = FALSE)
```

Comment:
The `integer64 = "character"` change is unrelated, but I recently did a complete reinstall of my R environment and profiles() threw this warning:

```
Warning messages:
1: In require_bit64_if_needed(ans) :
  Some columns are type 'integer64' but package bit64 is not installed. Those columns will print as strange looking floating point data. There is no need to reload the data. Simply install.packages('bit64') to obtain the integer64 print method and print the data again.
```

There is no real reason to have thread IDs as numeric/integer so this change seems more reasoanble then requiring bit64.

Reply:
Sounds good